src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsFloorAdhesionOldObject.java

```diff
@@ -6,14 +6,10 @@
 
 public class AdsFloorAdhesionOldObject extends AdsBaseObject {
 
-  private static final String FLOOR_ADHESION_CSS = "#ext-wikia-adEngine-template-floor";
-  private static final String
-      FLOOR_ADHESION_AD_FRAME_CSS
-      = "#ext-wikia-adEngine-template-floor .ad iframe";
-  private static final String FLOOR_ADHESION_IMAGE_IN_FRAME_CSS = "img";
+  private static final String FLOOR_ADHESION_CSS = "#ext-wikia-adEngine-template-floor, #invisible_high_impact_2";
   private static final String
       FLOOR_ADHESION_CLOSE_CSS
-      = "#ext-wikia-adEngine-template-floor .close";
+      = "#ext-wikia-adEngine-template-floor .close, .button-close";
```

Comment:
I believe we should have a bit more detailed selector, for example:
`#ext-wikia-adEngine-template-floor .close, .invisible-high-impact-wrapper .button-close`